### PR TITLE
Rules and automation enrolment

### PR DESCRIPTION
**Pull request by adeelq**

### CODE REVIEW

File: Helper/Automation.php [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Dotdigitalgroup\Email\Helper;
+
+class Automation extends \Magento\Framework\App\Helper\AbstractHelper
```
Comment (cdiacon) [inline Helper/Automation.php]: @adeelq New helper file??? call the model, it just not right to use it for one method and definitely is not going global/multiuse.

Comment (adeelq) [inline Helper/Automation.php]: @cdiacon created helper thought this method will be used at least two places. One observer and one plugin. You still think should move this to Model. I mean i'm fine with moving this to Model as well.

Comment (cdiacon) [inline Helper/Automation.php]: yes please move newCustomerAutomation to Automation model.

**Comment by adeelq** [comment]
@cdiacon suggested changes made

**Pull request merged**